pkg/metrics/schema.go

```diff
@@ -3,38 +3,40 @@ package metrics
+
+	// HTTPRequestMetricName - metric from http server.
+	HTTPRequestMetricName = "http_requests_total"
+	// HTTPRequestLatencyMetricName - metric from http server.
+	HTTPRequestLatencyMetricName = "http_requests_latency_seconds"
+	// HTTPErrorMetricName - metric from http server.
+	HTTPErrorMetricName = "http_errors_total"
+
+	// Circuit metrics.
+
+	// CircuitSignalReadingMetricName - used in circuit metrics.
+	CircuitSignalReadingMetricName = "circuit_signal_reading"
 	// FluxMeterMetricName name of fluxmeter metrics.
 	FluxMeterMetricName = "flux_meter"
+
+	// Workload metrics.
+
 	// WorkloadLatencyMetricName - metric used for grouping latencies per workload.
 	WorkloadLatencyMetricName = "workload_latency_ms"
-	// RequestCounterMetricName - metric from http server.
-	RequestCounterMetricName = "http_server_request_counter"
-	// ErrorCountMetricName - metric from http server.
-	ErrorCountMetricName = "http_server_error_counter"
-	// LatencyHistogramMetricName - metric from http server.
-	LatencyHistogramMetricName = "http_server_request_latency_seconds"
-	// AcceptedConcurrencyMetricName - metric for measuring latencies of accepted requests.
-	AcceptedConcurrencyMetricName = "accepted_concurrency"
-	// IncomingConcurrencyMetricName - metric for measuring latencies of all incoming requests.
-	IncomingConcurrencyMetricName = "incoming_concurrency"
 	// WorkloadLatencySumMetricName - metric from workload histogram.
 	WorkloadLatencySumMetricName = "workload_latency_ms_sum"
 	// WorkloadLatencyCountMetricName - metric from workload histogram.
 	WorkloadLatencyCountMetricName = "workload_latency_ms_count"
+
+	// AcceptedConcurrencyMetricName - metric for measuring latencies of accepted requests.
+	AcceptedConcurrencyMetricName = "accepted_concurrency_total"
+	// IncomingConcurrencyMetricName - metric for measuring latencies of all incoming requests.
+	IncomingConcurrencyMetricName = "incoming_concurrency_total"
+
 	// WFQFlowsMetricName - weighted fair queuing number of flows gauge.
-	WFQFlowsMetricName = "wfq_flows"
+	WFQFlowsMetricName = "wfq_flows_total"
 	// WFQRequestsMetricName - weighted fair queuing number of requests gauge.
-	WFQRequestsMetricName = "wfq_requests"
-	// FlowControlRequestsMetricName - counter for Check requests for flowcontrol.
-	FlowControlRequestsMetricName = "flowcontrol_requests_count"
-	// FlowControlDecisionsMetricName - counter for Check requests per decision type.
-	FlowControlDecisionsMetricName = "flowcontrol_decisions_count"
-	// FlowControlErrorReasonMetricName - metric for error reason on FCS Check requests.
-	FlowControlErrorReasonMetricName = "flowcontrol_error_reason_count"
-	// FlowControlRejectReasonMetricName - metric for reject reason on FCS Check requests.
-	FlowControlRejectReasonMetricName = "flowcontrol_reject_reason_count"
+	WFQRequestsMetricName = "wfq_requests_total"
 	// TokenBucketMetricName - a gauge that tracks the load shed factor.
 	TokenBucketMetricName = "token_bucket_lsf"
```

Comment:
Load shed factor is a `ratio`. Refer [prom unit conventions](https://prometheus.io/docs/practices/naming/)